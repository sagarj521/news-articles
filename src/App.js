import React, { Fragment } from 'react';
import './App.css';
import ArticleList from './containers/ArticleList';

function App() {
	return (
		<Fragment>
			<div className="App">
				<h1>NY Times Most Popular Articles</h1>
			</div>
			<ArticleList />
		</Fragment>
	);
}

export default App;
