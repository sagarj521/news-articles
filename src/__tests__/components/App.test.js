import { render, screen } from "@testing-library/react";
import App from "../../App";

test("renders app title", () => {
  render(<App />);
  const linkElement = screen.getByText(/NY Times Most Popular Articles/i);
  expect(linkElement).toBeInTheDocument();
});

test("matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
