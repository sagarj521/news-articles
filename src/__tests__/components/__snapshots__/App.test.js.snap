// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      NY Times Most Popular Articles
    </h1>
  </div>
  <ul
    class="article-list"
  >
    <div
      class="loading"
    />
  </ul>
</DocumentFragment>
`;
